src/runtime_src/core/include/xcl_graph.h

```diff
@@ -89,4 +94,22 @@ xclStopProfiling(xclDeviceHandle handle, int phdl);
 int
 xclLoadXclBinMeta(xclDeviceHandle handle, const xclBin *buffer);
 
+//currently adding below temporary functions to avoid too many changes in other dependency files, 
+//all these "xcl" functions will be cleaned up in next PR
+
+int
+xclSyncBOAIE(xclDeviceHandle handle, xrt::bo& bo, const char *gmioName, enum xclBOSyncDirection dir, size_t size, size_t offset, zynqaie::Aie* aie_array);
```

Comment:
We shouldn't be adding this functions at coreutil level. 
Lets discuss and close this offline.